### PR TITLE
Feat/open search

### PR DESCRIPTION
**Pull request by vvandijck**
This pull request includes:
- New `Opensearch` tab.

Note: I've tried to match the requirements from the tickets as much as possible, but in some cases it was cleaner to use a different way of representing the collected data.

### CODE REVIEW

File: src/heads-up/components/collapsible-link-list.vue [hunk 1]
```diff
@@ -0,0 +1,239 @@
+<template>
+  <div class="collapsible-list">
```
Comment (Jamerrone) [inline src/heads-up/components/collapsible-link-list.vue]: In my opinion, the component's main class should reflect the file name.

File: src/heads-up/lib/find-meta.js [hunk 2]
```diff
@@ -42,6 +42,35 @@ export function findMetaProperty(head, name) {
   return item ? item.content : null
 }
 
+export function findXMLElement(text, tagName) {
+  const parser = new DOMParser()
+  const xml = parser.parseFromString(text, 'text/xml')
+
+  const tags = Object.values(xml.documentElement.childNodes)
```
Comment (Jamerrone) [inline src/heads-up/lib/find-meta.js:49]: I am quite sure you can replace both `filters` with a single `find()`.

**Review by bob-voorhoede — APPROVED** [review]
✅

File: src/heads-up/lib/find-meta.js (hunk 2 above)
Comment (vvandijck) [reply src/heads-up/lib/find-meta.js:49]: I need an Array of `childNodes` that were found, and `find()` just returns the element.

**Review by Jamerrone — APPROVED** [review]

**Pull request merged**